cfy_manager/main.py

```diff
@@ -1088,6 +1088,8 @@ def upgrade(rpm=None, verbose=False, config_file=None):
         component.stop()
     set_globals()
     components.Prometheus().configure(upgrade=True)
+    components.Composer().configure()
+    components.Stage().configure()
```

Comment:
This won't work here since the DB is down. It should be run in before we stop the components.